pkg/server/server_test.go

```diff
@@ -593,6 +593,7 @@ func TestServerLoaduserDatabase(t *testing.T) {
 }
 
 func TestServerLoadUserDatabases(t *testing.T) {
+	os.RemoveAll("./data_v1.1.0")
```

Comment:
a tmp folder could be used here as well

Reply:
for some reason it doesn't work as expected with tmp folder